vpc.tf

```diff
@@ -33,63 +33,64 @@ module "vpc-label" {
 # VPC, SUBNETS AND ENDPOINTS DEPLOYED FOR FULLY PRIVATE EKS CLUSTERS
 # ---------------------------------------------------------------------------------------------------------------------
 module "vpc" {
-  create_vpc = var.create_vpc
-  source     = "terraform-aws-modules/vpc/aws"
-  version    = "v3.2.0"
-  name       = module.vpc-label.id
-  cidr       = var.vpc_cidr_block
-  azs        = data.aws_availability_zones.available.names
+  region               = data.aws_region.current.name
+  create_vpc           = var.create_vpc
+  source               = "aws-ia/vpc/aws"
+  version              = "0.1.1"
+  name                 = module.vpc-label.id
+  cidr                 = var.vpc_cidr_block
+  availability_zones   = data.aws_availability_zones.available.names
   # Private Subnets
-  private_subnets     = var.enable_private_subnets ? var.private_subnets_cidr : []
+  private_subnets_a   = var.enable_private_subnets ? var.private_subnets_cidr : []
   private_subnet_tags = var.enable_private_subnets ? local.private_subnet_tags : {}
 
   # Public Subnets
-  public_subnets     = var.enable_public_subnets ? var.public_subnets_cidr : []
-  public_subnet_tags = var.enable_public_subnets ? local.public_subnet_tags : {}
-
-  enable_nat_gateway = var.enable_nat_gateway ? var.enable_nat_gateway : false
-  single_nat_gateway = var.single_nat_gateway ? var.single_nat_gateway : false
-  create_igw         = var.enable_public_subnets && var.create_igw ? var.create_igw : false
```

Comment:
@jaymccon I think we need a feature in our VPC module to create a NAT and Internet gateway. Customers might using proxy servers instead of NAT or Internet gateway. However, This is a switch on/off feature and its helpful for workshops and demos